components/view-table/index.js

```diff
@@ -27,7 +29,9 @@ const ViewTable = ({ data, onEdit, onDelete }) => {
                       <span>{data[element]}</span>
                     )
                   ) : (
-                    <span>N/A</span>
+                    <span style={{ color: 'rgb(70,70,70)', fontVariant: 'small-caps' }}>
```

Comment:
I would recommend giving it a class instead of inline style here. We might use the same format elsewhere as well, such as in View (show)